server.js

```diff
@@ -2,19 +2,19 @@
  *  @license
  *  Copyright 2018-2021 MobiledgeX, Inc. All rights and licenses reserved.
  *  MobiledgeX, Inc. 156 2nd Street #408, San Francisco, CA 94105
- * 
+ *
```

Comment:
Are the changes to this file just spacing?

Reply:
Spacing and there is a new way of parsing urls, and passing the ports.